ui/nuxeo-i18n-behavior.js

```diff
@@ -139,4 +139,11 @@ export const I18nBehavior = {
     // this.i18n = null;
     this.set('i18n', window.nuxeo.I18n.translate);
   },
+
+  i18nWithPrefix(prefix, key, ...args) {
```

Comment:
Don't think this is needed elsewhere other than the format behavior, so I'd suggest moving it as a private method to the `nuxeo-format-behavior`(prefixed with `_`), perhaps with a different name, such as `_getTranslationForId(prefix, id)`.

Reply:
Agree!

Reply:
Updated!